node-notes-cli/data-delete.js

```diff
@@ -0,0 +1,11 @@
+const data = require('./data.json');
+
+function deleteEntry(id) {
+  for (var key in data.notes) {
+    if (id === key) {
+      delete data.notes[key];
+    }
+  }
```

Comment:
You can accomplish this and update without the `for ... in` loop by using bracket notation.
```js
delete data.notes[id];
```